pages/ksphere/kommander/latest/operations/access-control/index.md

```diff
@@ -6,36 +6,30 @@ menuWeight: 1
 excerpt: Centrally manage access across clusters
 ---
 
-Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters. The resources are similar to Kubernetes RBAC but with some crucial differences.
+Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters or a subset of clusters. The resources may seem familiar because they are similar to Kubernetes RBAC but with some crucial differences that make it possible to define the roles and policies once and have them federated to clusters within a given scope:
 
-## Types of Access Control Objects
-
-**Groups** map to groups and user claims from your identity providers.
-
-![Groups](/ksphere/kommander/img/Access-control-groups-table.png)
+- Global or Workspace-level roles and policies federate those resources to cluster-scoped roles and policies. In the case of workspace-level resources, they only apply to workspace clusters.
 
-Groups
+- Project-level roles and policies federate namespace-scoped role and policies to project clusters.
 
-**Roles** are named collections of rules defining which verbs can be applied to which resources.
+This approach gives you maximum flexibility over who has access to what resources, conveniently mapped to your existing identity providers' claims.
 
-![Roles](/ksphere/kommander/img/Access-control-roles-table.png)
-
-Roles
+## Types of Access Control Objects
 
-**Policies** bind a group to a role
+### Groups
 
-![Policies](/ksphere/kommander/img/Access-control-policies-table.png)
+You can map group and user claims made by your configured identity providers to Kommander groups by clicking the Groups tab under Global Administration / Identity providers.
```

Comment:
I think "You can map group and user claims made by your configured identity providers to Kommander groups by clicking the Groups tab under **Administration / Identity providers on a Global-level**." is clearer.